task0/LinearRegression_Ferhat.ipynb

```diff
@@ -0,0 +1,1853 @@
+{
```

Comment:
<p>Whenever you report sth to someone, keep in mind that your results should be grasped without much effort if you want to deliver your message clearly. It's always useful to divide them:</p><p> </p><p>ANSWERS:</p><p>1 - The score are as the following:</p><p>	A. ToNthHouse=10 -&gt; 0.752</p><p>	B. ToNthHouse=14 -&gt; 0.116</p><p> </p><p>2 - Outliers are:</p><p>	A. data(4)</p><p>	B. data(4, 10, 11, 12, 13)</p><p> </p><p> 3 - For situation A, linear regression fits to data set; but in situation B, we might need higher degree polynomial approach.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>